File: 0263-ugly-number/0263-ugly-number.go
func isUgly(n int) bool {
    isDivided := true
    for ;isDivided && n > 1;{
        isDivided = false
        for _, num := range []int{2,3,5} {
            if n%num == 0 {
                n = n/num
                isDivided = true
            }
        }
    }
    return n == 1
}

File: 0263-ugly-number/0263-ugly-number.py
class Solution:
    def isUgly(self, n: int) -> bool:
        divided = True
        while n>1 and divided:
            divided = False
            for num in (2, 3, 5):
                if not n%num:
                    n = n//num
                    divided = True
        return n == 1

        